Tribler/Test/test_my_channel.py

```diff
@@ -94,7 +111,7 @@ def do_add_torrent(torrentfilename):
             managefiles = self.managechannel.fileslist
             manager = managefiles.GetManager()
             manager.startDownload(torrentfilename, fixtorrent=True)
-            manager.startDownloadFromUrl(TORRENT_R, fixtorrent=True)
+            manager.startDownloadFromUrl(r'http://localhost:7286/video.torrent', fixtorrent=True)
```

Comment:
yes, or breakage will ensue when/if the port is busy (concurrent test runs, etc.)